roles/confluent.variables/defaults/main.yml

```diff
@@ -1101,27 +1106,383 @@ control_center_health_check_user: "{{control_center_ldap_user}}"
+kafka_connect_replicator_monitoring_interceptor_keystore_path: "/var/ssl/private/kafka_connect_replicator_monitoring_interceptor.keystore.jks"
+kafka_connect_replicator_monitoring_interceptor_truststore_storepass: "{{ ssl_truststore_password if kafka_connect_replicator_monitoring_interceptor_ssl_provided_keystore_and_truststore|bool else 'confluenttruststorepass'}}"
+kafka_connect_replicator_monitoring_interceptor_keystore_storepass: "{{ ssl_keystore_store_password if kafka_connect_replicator_monitoring_interceptor_ssl_provided_keystore_and_truststore|bool else 'confluentkeystorestorepass'}}"
+kafka_connect_replicator_monitoring_interceptor_keystore_keypass: "{{ ssl_keystore_key_password if kafka_connect_replicator_monitoring_interceptor_ssl_provided_keystore_and_truststore|bool else kafka_connect_replicator_monitoring_interceptor_keystore_storepass}}"
+kafka_connect_replicator_monitoring_interceptor_ca_cert_path: "/var/ssl/private/kafka_connect_replicator_monitoring_interceptor_ca.crt"
+kafka_connect_replicator_monitoring_interceptor_cert_path: "/var/ssl/private/kafka_connect_replicator_monitoring_interceptor.crt"
+kafka_connect_replicator_monitoring_interceptor_key_path: "/var/ssl/private/kafka_connect_replicator_monitoring_interceptor.key"
+kafka_connect_replicator_monitoring_interceptor_export_certs: /etc/security/keytabs/kafka_connect_replicator_monitoring_interceptor.keytab
+kafka_connect_replicator_monitoring_interceptor_keytab_path: /etc/security/keytabs/kafka_connect_replicator_monitoring_interceptor.keytab
+
+### Reconfiguration pattern. Set to parallel to reconfigure all hosts at once, which will cause downtime.
+deployment_strategy: serial
+
+### Zookeeper reconfiguration pattern. Set to parallel to reconfigure all hosts at once, which will cause downtime.
 zookeeper_deployment_strategy: "{{deployment_strategy}}"
 
-### Deployment strategy for Kafka. Set to parallel to run all provisionging tasks in parallel on all hosts, which may cause downtime.
+### Kafka reconfiguration pattern. Set to parallel to reconfigure all hosts at once, which will cause downtime.
 kafka_broker_deployment_strategy: "{{deployment_strategy}}"
 
-### Deployment strategy for Connect. Set to parallel to run all provisionging tasks in parallel on all hosts, which may cause downtime.
+### Connect reconfiguration pattern. Set to parallel to reconfigure all hosts at once, which will cause downtime.
 kafka_connect_deployment_strategy: "{{deployment_strategy}}"
 
-### Deployment strategy for Rest Proxy. Set to parallel to run all provisionging tasks in parallel on all hosts, which may cause downtime.
+### Rest Proxy reconfiguration pattern. Set to parallel to reconfigure all hosts at once, which will cause downtime.
 kafka_rest_deployment_strategy: "{{deployment_strategy}}"
 
-### Deployment strategy for ksqlDB. Set to parallel to run all provisionging tasks in parallel on all hosts, which may cause downtime.
```

Comment:
I'd like to keep the docs the way they were for my rolling deployment stuff. I suspect this is just from a merge